src/hooks/__tests__/useClickOutside.jest.ts

```diff
@@ -48,4 +48,27 @@ describe("useClickOutside", () => {
       return expect(callbackStub.mock.calls.length).toEqual(0);
     });
   });
+
+  describe("contextMenu - using mousedown", () => {
+    it("should not call the callback when click outside the element", () => {
+      act(() => {
+        fireEvent.mouseDown(document.body);
```

Comment:
this test the check for click use case so:
```suggestion
        fireEvent.click(document.body);
```

Reply:
`click` will work here because it's the default value, but `mouseDown` won't so Im checking this one. next test will check the same but with the new `eventName` passing